packages/react/scripts/generatePrimitivesCatalog.ts

```diff
@@ -0,0 +1,93 @@
+import fs from 'fs';
+import path from 'path';
+import { Project, Node, Symbol } from 'ts-morph';
+
+const TOO_MANY_STRINGS = 20;
+
+enum ComponentPropType {
+  Boolean = 'boolean',
+  String = 'string',
+  Number = 'number',
+  Any = 'any',
+}
+
+/**
+ * Determine if a TypeScript AST Node is a React.FC instance
+ */
+const isReactComponent = (node: Node) =>
+  Node.isVariableDeclaration(node) &&
+  node.getType().getText(node).startsWith('React.FC');
+
+/**
+ * Remove double quotes from String literal types
+ */
+const cleanStringLiteral = (literal: string) => literal.replace(/"/g, '');
+
+/**
+ * Filter browser vendor prefixes (-webkit, -moz, -ms, ...)
+ */
+const isValidStringValue = (value: string) => !value.startsWith('-');
```

Comment:
Curious where browser prefixes are being found in the Types

Reply:
Oh, I think these are included in the React CSS property types, aren't they?

Reply:
yep!